packages/action-menu/test/action-menu.test.ts

```diff
@@ -88,6 +90,42 @@ describe('Action menu', () => {
 
         await expect(el).to.be.accessible();
     });
+    it('loads - [custom icon]', async () => {
```

Comment:
Could we use this example in the `*.stories.ts` as well so that it's included in Storybook?

Reply:
Doing so means we can also include the name of the resulting story in `test/visual/stories.js` so that it is included in the visual regressions.